effekt/jvm/src/test/scala/effekt/core/TestRenamer.scala

```diff
@@ -0,0 +1,120 @@
+package effekt.core
+
+import effekt.{ core, symbols }
+import effekt.context.Context
+
+/**
+ * Freshens bound names in a given term for tests.
+ * Please use this _only_ for tests.
```

Comment:
Eventually, we could think about optimizing this renamer for tests slightly. E.g., choose indices that make diffs more readable (e.g. based on levels like the original one).
But fine for now :)

Reply:
```suggestion
 * Freshens bound names in a given term for tests.
 * Please use this _only_ for tests. Otherwise, prefer [[effekt.core.Renamer]].
```
Maybe make this more actionable.